### DISCUSSION TITLE
What's new: Playgrounds!

### DISCUSSION THREAD

**Discussion by robertaboukhalil** [sandbox-bio/sandbox.bio#12] [Announcements]
### What's new

- Interactive playgrounds for awk, grep, sed, jq: https://sandbox.bio/playgrounds?id=awk
- New tools in command line playground: `sed` for search & replace, `fasttree` for building phylogenetic trees

**Comment by ztgasdf** [comment]
Trying out the jq playground and it works with ease! One thing I want to ask is for the output box to wrap under the Flags area instead of staying on the same line as the Input area. I'd rather not have to scroll down every time to see the beginning of the output when I'm building a large filter.

**Reply by robertaboukhalil** [reply]
Great idea! I moved the Flags section to the left. Now the output should be visible even with large commands.
